massgen/frontend/displays/shared/tui_debug.py

```
@@ -0,0 +1,44 @@
+
+
+def get_tui_debug_logger() -> logging.Logger:
+    """Get or create TUI debug logger.
+
+    Returns:
+        Logger instance configured for TUI debugging.
+    """
+    logger = logging.getLogger("tui_debug")
+    if not logger.handlers:
+        # Create file handler
+        handler = logging.FileHandler("/tmp/tui_debug.log", mode="a")
```

Ruff (0.14.14)

[error] Probable insecure usage of temporary file or directory: "/tmp/tui_debug.log"

(S108)